gobblin-compliance/src/main/java/gobblin/compliance/HivePurgerPartitionRecordSchema.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.compliance;
+
+/**
+ * This is a dummy class required to pass in place of schema corresponding to {@link HivePurgerPartitionRecord}
+ *
+ * @author adsharma
+ */
+public class HivePurgerPartitionRecordSchema {
```

Comment:
I assume you are using this schema as input and output schemas in the converter. 
`public class HivePurgerConverter extends Converter<ComplianceRecordSchema, ComplianceRecordSchema, ComplianceRecord, ComplianceRecord>`.
You can change it to 
`public class HivePurgerConverter extends Converter<Class<?>, Class<?>, ComplianceRecord, ComplianceRecord>`

See `gobblin/gobblin-core/src/main/java/gobblin/converter/objectstore/ObjectStoreConverter.java`